server/src-exec/Migrate.hs

```diff
@@ -1,3 +1,9 @@
+{- |
+Description: Migrations for Hasura metadata.
```

Comment:
Should be `Hasura catalog`

catalog: The Postgres Schema
metadata: Stuff present in the catalog